crypto/benches/criterion_merkle.rs

```diff
@@ -8,29 +8,41 @@ use lambdaworks_math::{
     field::fields::fft_friendly::stark_252_prime_field::Stark252PrimeField,
 };
 use sha3::Keccak256;
+
 type F = Stark252PrimeField;
 type FE = FieldElement<F>;
-
 type TreeBackend = FieldElementBackend<F, Keccak256, 32>;
 
 fn merkle_tree_benchmarks(c: &mut Criterion) {
     let mut group = c.benchmark_group("Merkle Tree");
     group.sample_size(10);
     group.measurement_time(Duration::from_secs(30));
 
-    // NOTE: the values to hash don't really matter, so let's go with the easy ones.
+    // Large number of leaves benchmark
     let unhashed_leaves: Vec<_> = core::iter::successors(Some(FE::zero()), |s| Some(s + FE::one()))
-        // `(1 << 20) + 1` exploits worst cases in terms of rounding up to powers of 2.
```

Comment:
The comments explaining the benchmark were there for a reason. Is there a reason to remove them?